content/applications/hr/time_off/management.rst

```diff
@@ -0,0 +1,132 @@
+==========
+Management
+==========
+
+.. _time_off/approvals:
+
+Time off and allocation requests typically undergo an approval process before being granted.
+Requests either need one or two approvals, if any, depending on how the specific type of time off is
+configured. These are all housed under the :guilabel:`Management` section of the *Time Off*
+application.
+
+Only users who can approve allocation and time off requests have the :guilabel:`Management` section
```

Comment:
Ahhh, ok- I can see that. I was using "User" because that is the term for osmeone who can log into the system, as opposed to an 'employee'. I know I had an issue for the Frontdesk app with this- because only actual "Users" in the strict sense of the word (someone with Odoo access) can use the app, but all 'employees' can sign in with a badge- so I have been using the word "user" in that context, as anyone who can sign in and use the app. But I will change this moving forward.